SimpleWAWS/Code/Constants.cs

```diff
@@ -46,5 +46,9 @@ public static class Constants
         public const string DotNetRuntime = "dotnet";
         public const string JavaScriptRuntime = "node";
         public const string FunctionsRuntimeAppSetting = "FUNCTIONS_WORKER_RUNTIME";
+        public const string FunctionsVersionAppSetting = "FUNCTIONS_EXTENSION_VERSION";
+        public const string NodeDefaultVersionAppSetting = "WEBSITE_NODE_DEFAULT_VERSION";
+        public const string FunctionsVersion = "~2";
+        public const string MinNodeVersionForFunctions = "10.14.1";
```

Comment:
```        
public const string FunctionsVersion = "~2";
        public const string MinNodeVersionForFunctions = "10.14.1";
``` 
These could be SimpleSettings instead with the respective default values so we can update them later if needed

Reply:
Done